media/css/firefox/independent-splash.less

```diff
@@ -4,15 +4,6 @@
 
 @import "../sandstone/lib.less";
 
-@font-face {
```

Comment:
True, but if we change the share CTA in the future (e.g. not to use font-awesome) then this could inadvertently effect this page.